cms/models/Item.js

```diff
@@ -45,15 +50,16 @@ Item.add({
 		ref: "Speciality",
 		many: true,
 	},
-	resourceLinks: {
-		type: Types.Url, 
-		label: "Resource links",
-	},
 	UKMiComment: {
 		type: Types.Html,
 		wysiwyg: true,
 		label: "UKMi comment",
 	},
+	resourceLinks: {
+		type: Types.Html,
+		label: "Resource links",
+		wysiwyg: true,
```

Comment:
does a list of links need a text formatting toolbar? also why is it type html?

Reply:
It's the easiest way to add hyperlinks, through the wysiwyg editor. Looked into removing all buttons except hyperlink from the editor but can't be done for a single field.